distributed/worker.py

```diff
@@ -368,11 +388,10 @@ def get_data(self, stream, keys=None, who=None):
             self.digests['get-data-load-duration'].add(stop - start)
         start = time()
         try:
-            compressed = yield write(stream, msg)
-            yield close(stream)
```

Comment:
Scratch that. If I add the `close` back, a timeout error is raised in `test_client_with_scheduler`.